examples/AzureSDKDemoSwift/Source/BlobDownloadViewController.swift

```diff
@@ -140,22 +138,71 @@ class BlobTableViewController: UITableViewController, MSALInteractiveDelegate {
     private func reloadTableView() {
         DispatchQueue.main.async { [weak self] in
             self?.tableView.reloadData()
-            self?.refreshControl?.endRefreshing()
         }
     }
 
+    private func reloadTableView(forTransfer transfer: BlobTransfer) {
```

Comment:
Supposedly better for performance, but it causes some animation glitches that are very annoying.

Reply:
I removed this.